learning_paths/visualising_architecture/3_structurizr.md

```diff
@@ -0,0 +1,39 @@
+# Structurizr
+We recommend using the [C4 Model](https://c4model.com/) in conjunction with the [Structurizr DSL](https://github.com/structurizr/dsl) as a way to automatically generate the C4 diagrams from code. This allows for the Structurizr code to be kept in the same repository as the code of the system, making it easier to keep the two up to date together.
+
+A [Structurizr Lite quick start template](https://github.com/madetech/structurizr-template) has been created that you can use as a starting point in your own projects. 
+
+## Quick overview
+All the code to define the model, relationships and views are held in a single `workspace.dsl` file that Structurizr uses to build the visual diagams.
+![Structurizr workspace](/images/structurizr_workspace.jpg)
```

Comment:
In the diff previous this image is showing on the same line as the preceding sentence? Will the extra line help?

```suggestion

![Structurizr workspace](/images/structurizr_workspace.jpg)
```